# Migrator scripts which include the hardcoded relative path
# '../admin/' are modified to include the correct full path
#
Index: cherokee-0.9.0/contrib/06to07.py
===================================================================
--- cherokee-0.9.0.orig/contrib/06to07.py	2008-09-26 12:41:38.000000000 -0500
+++ cherokee-0.9.0/contrib/06to07.py	2008-09-26 12:42:04.000000000 -0500
@@ -9,7 +9,7 @@
 
 import sys
 
-sys.path.append('../admin/')
+sys.path.append('/usr/share/cherokee/admin/')
 from config import *
 
 def reparent_child (orig, target):
Index: cherokee-0.9.0/contrib/07to08.py
===================================================================
--- cherokee-0.9.0.orig/contrib/07to08.py	2008-09-26 12:41:38.000000000 -0500
+++ cherokee-0.9.0/contrib/07to08.py	2008-09-26 12:42:04.000000000 -0500
@@ -9,7 +9,7 @@
 
 import sys
 
-sys.path.append('../admin/')
+sys.path.append('/usr/share/cherokee/admin/')
 from config import *
 
 
Index: cherokee-0.9.0/contrib/08to09.py
===================================================================
--- cherokee-0.9.0.orig/contrib/08to09.py	2008-09-26 12:42:22.000000000 -0500
+++ cherokee-0.9.0/contrib/08to09.py	2008-09-26 12:42:27.000000000 -0500
@@ -9,7 +9,7 @@
 
 import sys
 
-sys.path.append('../admin/')
+sys.path.append('/usr/share/cherokee/admin/')
 from config import *
 
 def save_result (content, file):
